hub/templates/hub/area/_json_data.html

```diff
@@ -35,6 +35,15 @@
         {% endfor %}
         </body>
     </table>
+{% elif dataset.name.lower == "net zero details" %}
+        <p class="mt-2 lh-1">
```

Comment:
I think this section is one level too far indented. (Should be 4 spaces, not 8?)